_includes/layout/base/html-head.html

```diff
@@ -16,6 +16,8 @@
 {% if page.lang == 'bg' or page.lang == 'el' or page.lang == 'ko' or page.lang == 'hi' or page.lang == 'pl' or page.lang == 'sl' or page.lang == 'ro' or page.lang == 'ru' or page.lang == 'tr' or page.lang == 'uk' or page.lang == 'zh_CN' or page.lang == 'zh_TW' %}<link rel="stylesheet" href="/css/sans.css">{% endif %}
 <script type="text/javascript" src="/js/base.js"></script>
 {% if page.id != 'download' %}<script type="text/javascript" src="/js/main.js"></script>{% endif %}
+<script src="/js/jquery/jquery-1.11.2.min.js"></script>
```

Comment:
Good to hear caching and compression are enabled.

However, it's not just download size that matters.
While the file may be cached (never a guarantee), it's parsed and executed by the JS engine every time and takes up memory afterwards.

I realize bitcoin.org isn't a web application where performance is crucial, but why load anything that isn't required? Generally, not just jQuery. Even if you have super-fast internet, last-gen processors and huge SSD drive, I'd still try to deliver the smallest payload possible.

Also, keep in mind this is version 1.11.2 which dates back to December 2014. "Old" is an understatement.

TL;DR:
- I'd use a QR generator that's has no dependencies. If not complicated, I'd do this at build time rather than each time a users opens bitcoin.org.
- I'd use CSS animations and tricks like forcing repaints or `requestAnimationFrame` / `setInterval`. It's just a div sliding down after all.
- For XHR, I'd use the lightest possible wrapper for `XMLHttpRequest` object I could find, unless bitcoin.org already has one (which isn't jQuery).
- DOM "interrogation" hasn't required jQuery since IE8, just use native methods.

However, if you guys still want to keep jQuery, at the very least update it to latest.